airbyte-webapp/src/components/Modal/ModalBody.tsx

```diff
@@ -1,12 +1,18 @@
+import classnames from "classnames";
+
 import styles from "./ModalBody.module.scss";
 
 interface ModalBodyProps {
   maxHeight?: number | string;
+  padding?: boolean;
```

Comment:
A better name for the boolean be `padded` since `padding` may imply that I could pass the CSS style